docs/standalone/no_attestation.sh

```diff
@@ -16,9 +16,16 @@
 set -e
 
 # Create a public and private key pair.
-gpg --quick-generate-key --yes attestor@example.com
-gpg --armor --export attestor@example.com > gpg.pub
-gpg --armor --export-secret-keys attestor@example.com > gpg.priv
+GPG_OUTPUT="$(gpg --quick-generate-key --yes attestor@example.com)"
+
+# Save its fingerprint.
+KEY_FINGER_PRINT="$(echo $GPG_OUTPUT | sed -n 's/.*\([A-Z0-9]\{40\}\).*/\1/p')"
+if [ ${#KEY_FINGER_PRINT} -ne 40 ]; then echo "Error: key fingerprnt is not 40 characters." ; exit
```

Comment:
Why does the fingerprint need to be 40 chars? The error message as is would be confusing to users.

Reply:
A GPG key fingerprint should be 40 chars. This is just to safeguard in case sed/gpg behaves differently across platform/version. Will improve the msg to be more general.